src/WebUI/WorkLoads/OtherWorkloadsTable.tsx

```diff
@@ -168,21 +187,33 @@ export class OtherWorkloads extends BaseComponent<IOtherWorkloadsProperties, IOt
         return columns;
     }
 
-    private static _renderSetNameCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<ISetWorkloadTypeItem>, statefulSet: ISetWorkloadTypeItem): JSX.Element {
-        return BaseKubeTable.renderTwoLineColumn(columnIndex, tableColumn, statefulSet.name, OtherWorkloads._getSetType(statefulSet.kind), css(colDataClassName, "two-lines", "zero-left-padding"), "primary-text", "secondary-text");
+    private static _renderSetNameCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<ISetWorkloadTypeItem>, workload: ISetWorkloadTypeItem): JSX.Element {
+        return BaseKubeTable.renderTwoLineColumn(columnIndex, tableColumn, workload.name, OtherWorkloads._getSetType(workload.kind), css(colDataClassName, "two-lines", "zero-left-padding"), "primary-text", "secondary-text");
```

Comment:
there is some change we have to do this, may be you can check my PR, it could be same, i just did not use this function and wrote twoline cell.

Reply:
done